File: app/src/main/java/ru/nikitazhelonkin/coinbalance/data/api/coin/GasTrackerApiService.java
package ru.nikitazhelonkin.coinbalance.data.api.coin;


import io.reactivex.Single;
import retrofit2.http.GET;
import retrofit2.http.Path;
import ru.nikitazhelonkin.coinbalance.data.api.response.GasTrackerBalanceResponse;

public interface GasTrackerApiService {

    @GET("v1/addr/{address}")
    Single<GasTrackerBalanceResponse> balance(@Path("address") String address);

}

File: app/src/main/java/ru/nikitazhelonkin/coinbalance/data/api/coin/GasTrackerApiService.kt
package ru.nikitazhelonkin.coinbalance.data.api.coin


import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Path
import ru.nikitazhelonkin.coinbalance.data.api.response.GasTrackerBalanceResponse

interface GasTrackerApiService {

    @GET("v1/addr/{address}")
    fun balance(@Path("address") address: String): Single<GasTrackerBalanceResponse>

}
